Enterprise-CMCS/macpro-mdct-mfp
# MDCT-2813: Initiatives Dashboard (Empty View)

### Description

Initial setup of the Initiatives dashboard (the empty view) with its corresponding nested forms. 

`Not within the scope of this PR`
- Dashboard statusing
- Submitting those (4) nested forms

### Related ticket(s)


MDCT-2813

---
### How to test

- Log in as a state user
- Create a new Work Plan
- Navigate to the State-Specific Initiatives
- Create a new initiative
- Click `Edit` on that initiative

You should see four (4) pre-populated rows! ✨ 

### Important updates


 `wp.json` now has nested forms under the State-Specific Initiatives dashboard

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
